packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```diff
@@ -42,7 +42,5 @@
   "server_configuration_netboot_invalid": "Invalide",
   "server_configuration_netboot_auth_method": "Méthode d'authentification :",
   "server_configuration_netboot_auth_method_sshkey": "Authentification par clé SSH",
-  "server_configuration_netboot_auth_method_password": "Authentification par mot de passe",
```

Comment:
Be carefu server_configuration_netboot_auth_method and server_configuration_netboot_auth_method_sshkey are still in use